app/src/main/java/org/mifos/selfserviceapp/ui/fragments/BeneficiaryApplicationFragment.java

```diff
@@ -0,0 +1,223 @@
+package org.mifos.selfserviceapp.ui.fragments;
+
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.AdapterView;
+import android.widget.ArrayAdapter;
+import android.widget.EditText;
+import android.widget.LinearLayout;
+import android.widget.Spinner;
+
+import org.mifos.selfserviceapp.R;
+import org.mifos.selfserviceapp.models.beneficary.Beneficiary;
+import org.mifos.selfserviceapp.models.beneficary.BeneficiaryPayload;
+import org.mifos.selfserviceapp.models.beneficary.BeneficiaryUpdatePayload;
+import org.mifos.selfserviceapp.models.templates.beneficiary.AccountTypeOption;
+import org.mifos.selfserviceapp.models.templates.beneficiary.BeneficiaryTemplate;
+import org.mifos.selfserviceapp.presenters.BeneficiaryApplicationPresenter;
+import org.mifos.selfserviceapp.ui.activities.base.BaseActivity;
+import org.mifos.selfserviceapp.ui.enums.BeneficiaryState;
+import org.mifos.selfserviceapp.ui.fragments.base.BaseFragment;
+import org.mifos.selfserviceapp.ui.views.BeneficiaryApplicationView;
+import org.mifos.selfserviceapp.utils.Constants;
+import org.mifos.selfserviceapp.utils.Toaster;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.inject.Inject;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+import butterknife.OnClick;
+
+/**
+ * Created by dilpreet on 16/6/17.
+ */
+
+public class BeneficiaryApplicationFragment extends BaseFragment implements
```

Comment:
@therajanmaurya this fragment allows to add and update a beneficiary due to which I named it BeneficiaryApplicationFragment